pages/contribute.tsx

```diff
@@ -19,12 +33,140 @@ function Contribute(): JSX.Element {
             site_name: 'open source at ACM at UCLA',
           }}
         />
-        <h1>
-          get started
-        </h1>
+        <div className='row spaced-row'>
+          <h1>get started </h1>
+          <a href='#good-first-issues'><button>good first issues</button></a>
```

Comment:
This seems like wrong markup?

Reply:
Do you mean with the space in the h1 or the button being in the same row as the h1? At first I had the button to the GFI projects on the same line as the header but I'll change that to point to a new page now.

Reply:
Ah no, in this case I mean having a button inside the anchor. I know it's tied to CSS but it's an accessibillity no-no.

(I know it's a little hypocritical since I banged out the first iteration of this code)